src/libdrakvuf/vmi.c

```diff
@@ -271,7 +270,12 @@ event_response_t post_mem_cb(vmi_instance_t vmi, vmi_event_t *event) {
     }
 
 done:
-    g_free(pass->procname);
+    if ( pass->proc_data )
+    {
+        if ( pass->proc_data->name )
```

Comment:
No need to check whether it's non-NULL, `g_free` is NULL safe.

Reply:
Agreed